### PR TITLE
Add frozen 1.0.0

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **frozen/1.0.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/frozen/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import os.path
+
+
+class FrozenConan(ConanFile):
+    name = "frozen"
+    description = "A header-only, constexpr alternative to gperf for C++14 users."
+    homepage = "https://github.com/serge-sans-paille/frozen"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("gperf")
```
Comment (uilianries) [inline recipes/frozen/all/conanfile.py:11]:
```suggestion
    topics = ("gperf", "constexpr", "header-only")
```

File: recipes/frozen/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    exports_sources = ["CMakeLists.txt"]
+    settings = "compiler"
+    generators = "cmake"
```
Comment (uilianries) [inline recipes/frozen/all/conanfile.py:14]:
```suggestion
    no_copy_source = True
```

File: recipes/frozen/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
```
Comment (uilianries) [inline recipes/frozen/all/conanfile.py:16]:
```suggestion
```

File: recipes/frozen/all/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    def source(self):
```
Comment (uilianries) [inline recipes/frozen/all/conanfile.py:18]:
```suggestion
    def configure(self):
        minimal_cpp_standard = "14"
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, minimal_cpp_standard)

        minimal_version = {
            "gcc": "5",
            "clang": "3.4",
            "apple-clang": "10",
            "Visual Studio": "14"
        }

        compiler = str(self.settings.compiler)
        if compiler not in minimal_version:
            self.output.warn(
                "%s recipe lacks information about the %s compiler standard version support" % (self.name, compiler))
            self.output.warn(
                "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
            return

        version = tools.Version(self.settings.compiler.version)
        if version < minimal_version[compiler]:
            raise ConanInvalidConfiguration(
                "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))

    def source(self):
```

**Comment by SSE4** [comment]
@uilianries I am surprised you didn't put frozen picture here:)

**Comment by Croydon** [comment]
giphy

**Comment by uilianries** [comment]
@SSE4 I'm a fan of animations, but I didn't watch Frozen yet

**Pull request merged**